client/src/components/MainPage/TaskModal/PriorityMenu/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import * as S from './style';
+
+const ReviewMenu = () => {
```

Comment:
중요도를 선택하는 부분인가요? 변수명이 명확하지 않은 것 같습니다
```suggestion
const ImportanceInput = () => {
```

Reply:
폴더명을 따라가도 좋을 것 같습니다~

Reply:
넵 수정했습니다